src/components/InlineMessage/InlineMessage.tsx

```diff
@@ -33,8 +32,9 @@ interface BannerTypographyProps {
   color: string;
 }
 
-const STypography = styled(Typography)<BannerTypographyProps>`
+const STypography = styled(Body)<BannerTypographyProps>`
   color: ${(props) => props.color};
+  vertical-align: middle;
```

Comment:
use inline styling instead of `styled-component`

Reply:
It complains when I use `verticalAlign="middle"` :(